src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Invitations/InvitationsControllerNegativeTests.cs

```diff
@@ -263,14 +264,20 @@ public async Task CreateInvitation_AsViewer_ShouldReturnForbidden_WhenPermission
 
 
         [TestMethod]
-        public async Task CreateInvitation_AsPlanner_ShouldReturn_SpecificError_WhenMailProblem()
+        public async Task CreateInvitation_AsPlanner_ShouldReturn_SpecificError_WhenFallbackProblem()
```

Comment:
This test don't belong here: It should be an unit test in CreateInvitationCommandHandlerTests

Reply:
Already handled by HandleCreateInvitationCommand_ShouldRollbackIfFusionApiAndICSFails in CreateInvitationCommandHandlerTests. Hence removing CreateInvitation_AsPlanner_ShouldReturn_SpecificError_WhenFallbackProblem from InvitationsControllerNegativeTest.